packages/pharos/src/components/breadcrumb/pharos-breadcrumb.wc.stories.mdx

```diff
@@ -22,7 +22,7 @@ export const Template = (args) =>
 
 # Breadcrumbs
 
-<pharos-link href="https://www.jstor.org/pharos/components/breadcrumb" target="_blank">See guidelines in Pharos</pharos-link>
+<pharos-link href="https://pharos.jstor.org/components/breadcrumb" target="_blank">See guidelines in Pharos</pharos-link>
```

Comment:
Seems we could extract this into a variable of some kind to avoid future churn of this magnitude

Reply:
Yeah I'm thinking we can create a component that returns the Pharos link